### DISCUSSION TITLE
realtime suddenly stopped working

### DISCUSSION THREAD

**Discussion by EdgarsLv** [supabase/supabase#10443] [Questions]
Anyone have issues with realtime?

Mine suddenly stoped working. Production app is affected. Getting this error for every realtime attempt.
And yes i wrote to support already.

**Comment by EdgarsLv** [comment]
Will bump this up. 1st time support fixed it pretty fast. 
But again realtime suddenly stopped working, but now with different error. This is just not seriously...

It's really only me or someone else have encountered realtime issues.
This is with supabase v2. 
For almost 1 year didn't had any issues with supabase v1.

**Comment by esDotDev** [comment]
I've also had this happen, my tables were working fine for a couple of weeks during development, now I get no events at all when tables are changed on the server.

**Reply by GaryAustin1** [reply]
The inspector here https://realtime.supabase.co/ is useful for verifying your instance's realtime status.
